app/lib/task/handlers.dart

```diff
@@ -42,6 +42,7 @@ Future<shelf.Response> handleDartDoc(
   shelf.Request request,
   String package,
   String version,
+  String segment,
```

Comment:
I see how we need to pass this around, but we need to call it something else.

`segment` is a name nobody can guess what is.
If we can't come up with a self-explanatory name, then we should document it.


We could also make this a `bool hasLatestInUrl`, right?
Because `segment == version || segment == 'latest'`, right? (I could be wrong here?, there might be other cases where this isn't the case, in particular when caching a redirect).